web/src/tippyjs.ts

```diff
@@ -504,6 +504,64 @@ export function initialize(): void {
         },
     });
 
+    tippy.delegate("body", {
+        target: ".add_member_button_wrapper",
+        onShow(instance) {
+            const $wrapper = $(instance.reference);
+            const $button = $wrapper.find("button");
+            const $container = $wrapper.closest(".add_members_container");
+
+            const button_is_disabled = Boolean($button.prop("disabled"));
+            const container_is_enabled = !$container.hasClass("add_members_disabled");
+
+            if (button_is_disabled && container_is_enabled) {
```

Comment:
A better condition to show this tooltip can be to just check if the input is empty or not. Can we do that?

Reply:
Ok. Then also maybe condition like `$input.val().length === 0 && container_is_enabled` would be better as the text shown is eventually for empty input.

Reply:
Ohh, I missed about checking the pills. Current condition seems fine then.